What's-new-in-TypeScript.md

```diff
@@ -82,6 +82,41 @@ let combined = combine(animalFunc, dogFunc);  // (x: Dog) => void
 Above, all inferences for `T` originate in contravariant positions, and we therefore infer the *best common subtype* for `T`.
 This contrasts with inferences from covariant positions, where we infer the *best common supertype*.
 
+## Support for JSX Fragment Syntax
+
+TypeScript 2.6.2 adds support for the new `<>...</>` syntax for fragments in JSX.
+It is frequently desirable to return multiple JSX elements from a component, as in the following scenario:
+
+```tsx
+render() {
+    return (
+        Some text.
+        <h2>A heading</h2>
+        More text.
+    );
+}
+```
+Since this is not a single element, this gives an error, so a common solution would be to wrap the text in an extra element, such as a `<div>` or `<span>`.
+
+To address this pattern, React introduced the `React.Fragment` component, which provides a dedicated way to wrap such elements without adding an element to the DOM.
+Correspondingly, the `<>...</>` syntax was added to JSX to facilitate this new construct. Therefore, the above scenario becomes:
+```tsx
+render() {
+    return (
+        <>
+            Some text.
+            <h2>A heading</h2>
+            More text.
+        </>
+    );
+}
+```
+
+Under `--jsx preserve`, the new syntax is left untouched for TypeScript emit. Otherwise, for `--jsx react`, `<>...</>` is compiled to `React.createElement(React.Fragment, null, ...)`, where `React.createElement` respects `--jsxFactory`.
+Note that due to inconsistent non-React framework support for fragments, `<>...</>` produces an error when `--jsx react` and `--jsxFactory` are used together.
```

Comment:
I do not think you need to mention this. if you must, just say it is an error to use fragments with `--jsxFactory`